### DISCUSSION TITLE
Security implications of using fixuid in entrypoint.sh?

### DISCUSSION THREAD

**Discussion by echuber2** [coder/code-server#2116] [Q&A] [answered]
I notice that you're using `fixuid` in [entrypoint.sh](https://github.com/cdr/code-server/blob/master/ci/release-image/entrypoint.sh) to adjust for differ user/group settings when the container is run. I'm just wondering what the security implications might be for that, since the `fixuid` documentation warns not to use it in production. Is this not a concern for persisting user mounts as long as users are trusted?

Thanks!

**Comment by code-asher** [comment, marked as answer]
I believe there aren't any known security issues: https://github.com/boxboat/fixuid/issues/1.

But if you're able to get away without using fixuid it makes sense to go ahead and do that.
